Gems/PhysX/Code/Tests/EditorTestUtilities.cpp

```diff
@@ -49,12 +51,17 @@ namespace PhysXEditorTests
         const AZ::Transform& transform,
         const AZ::Vector3& nonUniformScale,
         const AZ::Vector3& boxDimensions,
-        const AZ::Vector3& translationOffset)
+        const AZ::Vector3& translationOffset,
+        bool dynamic)
```

Comment:
nit: I'm not a fan of bool parameters (as you well know 😅) as the callsites are always hard to read and it's all too easy for them to multiply. I'd potentially suggest making this a named enum (or using an existing one?) to make the call easier to read (`Static` and `Dynamic` perhaps) or provide a callback or component list for the user to customize the Editor Entity and add other types of components to make this more flexible (and still easier to read at the callsite)

Reply:
Switched to an enum.